ovoenergy/ovoenergy.py

```diff
@@ -97,7 +97,7 @@ async def get_daily_usage(self, month: str) -> OVODailyUsage:
 
         if "gas" in json_response:
```

Comment:
This line should already cover no gas object in the response

Reply:
No it doesn't. "gas" comes back as None. So it is present on the response object but it equal to None, and therefore the next if statement fails.

Reply:
`{'electricity': {'data': [...], 'next': False, 'prev': True, 'prevYear': True}, 'gas': None}`